client/src/helpers/APICalls/createSchedule.ts

```diff
@@ -0,0 +1,57 @@
+import { AuthApiData } from '../../interface/AuthApiData';
+import { FetchOptions } from '../../interface/FetchOptions';
+
+const createSchedule = async (data: {
+  name: string;
+  days: {
+    monday: {
+      active: boolean;
```

Comment:
Why don't we use the `DayInformation` interface to avoid repeating ourselves here

Reply:
Done the changes

Reply:
Done changes